src/Private/Console.ps1

```diff
@@ -0,0 +1,987 @@
+    Write-PodeHost ']              ' -ForegroundColor $headerColor -Force:$Force -NoNewLine:$noHeaderNewLine
+
+    if ($headerSeparator) {
+        # Write a horizontal divider line to the console.
+        Write-PodeHostDivider -Force $true
+    }
+
+    if ($serverState -eq 'Running') {
+        if ($PodeContext.Server.Console.ShowEndpoints) {
+            # state what endpoints are being listened on
+            Show-PodeConsoleEndpointsInfo -Force:$Force
+        }
+        if ($PodeContext.Server.Console.ShowOpenAPI) {
+            # state the OpenAPI endpoints for each definition
+            Show-PodeConsoleOAInfo -Force:$Force
+        }
+    }
+
+    if ($showHelp) {
+        Show-PodeConsoleHelp
+    }
+    elseif ($ctrlH ) {
+        Show-PodeConsoleHelp -Hide
+    }
+
+    if ($footerSeparator) {
+        # Write a horizontal divider line to the console.
+        Write-PodeHostDivider -Force $true
+    }
+}
+
+
+<#
+.SYNOPSIS
+    Displays or hides the help section for Pode server control commands.
+
+.DESCRIPTION
+    The `Show-PodeConsoleHelp` function dynamically displays a list of control commands available for managing the Pode server.
+    Depending on the `$Hide` parameter, the help section can either be shown or hidden, with concise descriptions for each command.
+    Colors for headers, keys, and descriptions are customizable via the `$PodeContext.Server.Console.Colors` configuration.
+
+.PARAMETER Hide
+    Switch to display the "Show Help" option instead of the full help section.
+
+.NOTES
+    This function is designed for Pode's internal console display system and may change in future releases.
+
+.EXAMPLE
+    Show-PodeConsoleHelp
+
+    Displays the full help section for the Pode server.
+
+.EXAMPLE
+    Show-PodeConsoleHelp -Hide
+
+    Displays only the "Show Help" option instead of the full help section.
+
+#>
+function Show-PodeConsoleHelp {
+    param(
+        [switch]
+        $Hide
+    )
+    # Retrieve centralized key mapping for keyboard shortcuts
+    $KeyBindings = $PodeContext.Server.Console.KeyBindings
+
+    # Define help section color variables with fallback defaults
+    $helpHeaderColor = if ($null -ne $PodeContext.Server.Console.Colors.HelpHeader) {
+        $PodeContext.Server.Console.Colors.HelpHeader
+    }
+    else {
+        [System.ConsoleColor]::Yellow
+    }
+
+    $helpKeyColor = if ($null -ne $PodeContext.Server.Console.Colors.HelpKey) {
+        $PodeContext.Server.Console.Colors.HelpKey
+    }
+    else {
+        [System.ConsoleColor]::Green
+    }
+
+    $helpDescriptionColor = if ($null -ne $PodeContext.Server.Console.Colors.HelpDescription) {
+        $PodeContext.Server.Console.Colors.HelpDescription
+    }
+    else {
+        [System.ConsoleColor]::White
+    }
+
+    $helpDividerColor = if ($null -ne $PodeContext.Server.Console.Colors.HelpDivider) {
+        $PodeContext.Server.Console.Colors.HelpDivider
+    }
+    else {
+        [System.ConsoleColor]::Gray
+    }
```

Comment:
`Protect-PodeValue` for these? Also worth checking other places in this file as well, as `Write-PodeHostDivider` and `Show-PodeConsoleEndpointsInfo` also have places that could use this too

Reply:
I remove them